src/metafile.c

```diff
@@ -1251,7 +1251,63 @@ update_emf_header (MetafileHeader *header, BYTE* data, int length)
 	return status;
 }
 
-static GpStatus 
+GpStatus
+gdip_read_emf_header_optionals (ENHMETAHEADER3 *header, void *pointer, ImageSource source)
```

Comment:
This algorithm is taken from https://msdn.microsoft.com/en-us/library/cc230635.aspx

Reply:
could you please put that link into the source?